training-react/tic-tac-toe/src/app/components/App/components/Game/components/Moves/components/Step/step.js

```diff
@@ -0,0 +1,25 @@
+import React, { PureComponent } from 'react';
+import { func, number } from 'prop-types';
+
+class Step extends PureComponent {
+  handleClick = () => {
+    const { onClick, move } = this.props;
+    onClick(move);
+  };
+
+  render() {
+    const { move } = this.props;
+    return (
+      <li key={move}>
```

Comment:
The `key` should be put where you do the `map` in this case in the `renderMoves` function in the `Moves` component. Also remember that the `key` shouldn't be the array index.